src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -72,84 +70,6 @@ void GcsResourceManager::HandleGetResources(const rpc::GetResourcesRequest &requ
   ++counts_[CountType::GET_RESOURCES_REQUEST];
 }
 
-void GcsResourceManager::UpdateResources(
```

Comment:
NOTE: As I mentioned in the PR's description, both `UpdateResources` and `DeleteResources` could be removed.